blobfuse/tests.py

```diff
@@ -53,6 +53,18 @@ def test_WriteReadSingleFile(self):
         os.remove(filepath)
         self.assertEqual(False, os.path.exists(filepath))
 
+    def test_WriteReadSingleFileUnicode(self):
+        file1txt = "你好，世界！"
```

Comment:
can you please check special characters as well? someone may like filenames like hello\world&we^are%all~together=Gagarin?perestroika and so on -)?